src/Riok.Mapperly/Descriptors/MappingBodyBuilders/NewInstanceObjectMemberMappingBodyBuilder.cs

```diff
@@ -66,6 +66,19 @@ private static void BuildInitOnlyMemberMappings(INewInstanceBuilderContext<IMapp
 
             BuildInitMemberMapping(ctx, targetMember, sourceMemberPath);
         }
+
+        // check for any configured mappings that match the ignored targets
+        // this way explicit property mapping will map ignored members
+        var initOnlyIgnoredTargetMembers = includeAllMembers
+            ? ctx.IgnoredTargetMembers.Values.ToArray()
+            : ctx.IgnoredTargetMembers.Values.Where(x => x.CanOnlySetViaInitializer()).ToArray();
+        foreach (var targetMember in initOnlyIgnoredTargetMembers)
+        {
+            if (ctx.MemberConfigsByRootTargetName.Remove(targetMember.Name, out var memberConfigs))
+            {
+                BuildInitMemberMapping(ctx, targetMember, memberConfigs);
+            }
+        }
```

Comment:
Instead of this, wouldn't it be much simpler to just add `ignoredTargetMemberNames.ExceptWith(MemberConfigsByRootTargetName.Keys);` in `MembersMappingBuilderContext` before initialising `IgnoredTargetMembers` (L45)?

Reply:
It would save time by removing ignored targets that are not explicitly mapped. I don't think it would replace this logic tho